canonical-web-and-design/canonical.com
# Implement career gamification

## Done
- Add `/careers/start` page & implement gamification

## QA
- Go to [/careers/start](http://0.0.0.0:8002/careers/start) 
- Compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1562)
- Make sure you can add and remove skills to the list at the top.
- Make sure you cannot select more than 5 skills
- Make sure a message is displayed when you click the submit button at the top

## Issue

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1562
